### DISCUSSION TITLE
Is Solana Token Extension Metadata displayed in Phantom wallet?

### DISCUSSION THREAD

**Discussion by romanix** [phantom/docs#184] [General]
Is Solana Token Extension Metadata somewhere displayed in Phantom wallet?
Let's say a token represented a public share with attributes like: 

- ISIN 
- FIGI
- Security Type
- Currency 

Would Phantom wallet display these attributes in the token details / about?
Is there an example of a token that has token2022 metadata that is shown in Phantom wallet? (URL?)

**Comment by adamdelphantom** [comment]
Hello @romanix,

Yes Phantom does support the token metadata extension. You will find an example [here.](https://docs.phantom.app/developer-powertools/solana-token-extensions-token22#metadata-extension)

**Reply by romanix** [reply]
Thank you for the response and the URL to an example! That helps
